packages/datadog-plugin-couchbase/test/integration-test/server.mjs

```diff
@@ -0,0 +1,17 @@
+import 'dd-trace/init.js'
+import { connect } from 'couchbase'
+
+async function main () {
```

Comment:
I believe this can be inlined in the root of the file? The other .mjs tests have a lot of top level awaits.